ckanext/dataset_subscriptions/templates/dataset-subscriptions_email_body.j2

```diff
@@ -1,8 +1,18 @@
-{% set num = activities|length %}
 {{ _("Here's a list of recently updated datasets you're following in DMS:") }}
 
-{% for activity in activities %}
-  {{ _("{count}. Dataset title: {title}").format(count=loop.index, title=activity.data.package.title) }}
+{% for activity in activities -%}
+  {% if activity.dataset_groups is defined -%}
+    {% set group_name = activity.dataset_groups[0].title + ' | ' -%}
+  {% else -%}
+    {% set group_name = "" -%}
+  {% endif -%}
+  {% if activity.dataset_extras is defined -%}
+    {% set program_area = activity.dataset_extras[0].value + ' | ' -%}
```

Comment:
you should make sure that the first value of dataset extras is actually the program_area.
Also this logic should be moved up to the `add_dataset_name_to_activity_list`. (also it might make sense to rename the method to `add_dataset_details_to_activity_list`.